packages/openapi-request-validator/index.ts

```diff
@@ -314,12 +357,18 @@ function extendedErrorMapper(mapper) {
 
 function getSchemaForMediaType(
   contentTypeHeader: string,
-  requestBodySpec: OpenAPIV3.RequestBodyObject
+  requestBodySpec: OpenAPIV3.RequestBodyObject,
+  logger: Logger
 ): string {
+  if (!contentTypeHeader) {
```

Comment:
why are we adding this without a test?

Reply:
Test if a logger gets passed? How can anything fail there? Or you're asking about `if (!contentTypeHeader)`? That's just refactoring, I've moved the conditional from `validate` because it makes more sense to check it here (so that every possible caller of getSchemaForMediaType doesnt have to perform this check), and it makes the code look better:
```
      const mediaTypeMatch =
        contentType && getSchemaForMediaType(contentType, this.requestBody);
``` 
becomes:
```
      const mediaTypeMatch = getSchemaForMediaType(contentType, this.requestBody);
```

Reply:
sorry for the late reply, i mean the `if` condition.  can you add a test that covers that?